main/static/src/style/test.scss

```diff
@@ -0,0 +1,48 @@
+.test {
+  section {
+    padding: $line-height-computed 0;
+    > h2 {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3